app/src/js/server/util.ts

```diff
@@ -81,7 +81,15 @@ function validateCognitoConfig (cognito: CognitoConfig | undefined) {
  *
  * @param {ServerConfig} config
  */
-function validateConfig (config: ServerConfig) {
+async function validateConfig (config: ServerConfig) {
+  if (!(await fs.pathExists(config.data))) {
```

Comment:
`data` may not exist on fs when s3 is used

Reply:
resolved